mteb/tasks/Classification/ssw/SiswatiNewsClassification.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 2800
+
+
+class SiswatiNewsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SiswatiNewsClassification",
+        description="Siswati News Classification Dataset",
+        reference="https://huggingface.co/datasets/dsfsi/za-isizulu-siswati-news",
+        dataset={
+            "path": "isaacchung/siswati-news",
+            "revision": "f5502326c4e48adc99b18b1582f68b8fb5e7ec30",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["zul-Latn"],
```

Comment:
```suggestion
        eval_langs=["ssw-Latn"],
```
Should it be ssw for the language code?